src/lib/Canvas.js

```diff
@@ -213,16 +213,36 @@ Canvas.prototype = {
         this.stopResizing();
     },
 
+    divBoundingClientRect: function() {
+        // Make sure our canvas has integral dimensions
+        var size = this.div.getBoundingClientRect();
+        var top = Math.floor(rect.top),
+            left = Math.floor(rect.left),
+            width = Math.ceil(rect.width),
+            height = Math.ceil(rect.height);
```

Comment:
did you try `Math.round` first rather than `.floor` & `.ceil` ?

Reply:
Round isn't quite right - i need to always round down for the top and left, and always round up for the bottom and right.